rfc/website-revamp.md

```diff
@@ -0,0 +1,115 @@
+* Improve operator, end-user, and contributor experience by providing more navigable documentation.
+* Improve content organization to facilitate easy contribution.
+* Improve local build time and adopt the new way GitHub pages uses themes
+
+### Non-Goals
+
+* Not a complete reorganization of the documentation set. What will happen is the current documentation related tabs will be condensed under a single “Docs” tab in the top navigation.
+* Not replatforming or rethemeing .io. The cost/benefit is not there to do this at this time.
+
+## Motivation and Rationale
+
+Spinnaker.io has operated with one basic design look/feel since March 22, 2017. Given the current momentum and size of the project, it is time for a refresh. The Docs SIG feels it’s time to evolve .io beyond a documentation site and into a project hub, providing a presence for Spinnaker that is clearly segmented for different user personas.
+
+We need to provide context to attract new companies and ICs as they encounter Spinnaker. This aligns with Travis Tomsu’s efforts to add more context and persuasive oomph to READMEs across the project. The window for Spinnaker inevitability is threatened by a crowded marketplace and will close; this project will boost Spinnaker’s profile and chances for inevitability.
+
+From a technical standpoint, the way GitHub pages manages themes has changed to remote themes, and our implementation is outdated. Relying on the Minimal Mistakes theme-maintainer’s repository results in less theme-related maintenance for the Docs SIG and capture of theme feature improvements.
+
+## Timeline
+* Achieve buy-in from stakeholders by March.
+* Finalize Statement of Work and share with community in March.
+* Finalize SIG-validated high-level design in April.
+* Sitemap
+* Accept contractor bids, including billing rate and time estimate, in April.
+* Identify and hire webdev/design contractor in April-May.
+* Work begins in May, validated with Docs SIG continuously.
+* Work completed by July 1.
+
+## Design 
+
+(In progress)
+
+### Top Nav Menu
+The current top nav will be condensed into 5 tabs:
+
+* Getting Started
+    * Basic conceptual info about Spinnaker for multiple personas.
+    * Direct people to the next logical step for their persona
+* Docs
+    * All the current Spinnaker product documentation
+* Community
+    * Contributor Experience Resource, such as code and documentation contribution guide and contribution best practices and tips
+    * Community Calendar
+    * Usage stats
+    * Governance & SIG info. This should be autogenerated from the Governance repo if possible. If not, we will continue to link to the Governance repo.
+* News
+    * Spinnaker newsletter
+    * Featured videos
+* Blog
+
+### The refreshed home page should:
+* Answer the question “Why Spinnaker” at high level and provide easy links to segmented answers (dev, c-level, director, partner etc)
+  * Individual Contributor: What does my life look like if I use Spinnaker?
```

Comment:
Is `Individual Contributor` here meant to be `End User`? Perhaps it would be worthwhile to add sections that:

1. More explicitly define the personas you reference in this RFC, and
2. More explicitly define the new entryways and paths that correspond to each persona.

From my reading of this RFC, we are considering the following personas:
- Individual Contributor: I am a developer who contributes code to the core Spinnaker project, or maintains a fork or custom extensions of Spinnaker for my company.
- Operator: I manage a Spinnaker installation for my company.
- End User: I am on an application team and use Spinnaker to manage my continuous delivery workflow.
- Non-technical stakeholder/C-level: I am evaluating Spinnaker as a means to achieve some business objective.
- Evaluator: I am currently evaluating Spinnaker for my company. I could be either an operator, end-user, or higher-level less technical stakeholder. This is almost a dimension of the other personas rather than its own persona, but since the first item in the `Goals` section refers to a persona of `someone brand new to Spinnaker`, might be worth nailing down which "new" personas we are targeting here.

Reply:
Thanks for the feedback @maggieneterval! We'll work on adding this info to the RFC. Some of it already exists in our notes.